archivebox/config.py

```diff
@@ -397,6 +398,55 @@ def get_commit_hash(config):
     except Exception:
         return None
 
+def get_version_releases(config):
+    """
+    returns a dictionary containing the GitHub release data for 
+    the recommended upgrade version and the currently installed version
+    """
+    github_releases_api = "https://api.github.com/repos/pirate/archivebox/releases"
+    response = requests.get(github_releases_api)
+    if response.status_code != 200:
+        stderr('Failed to get release data from GitHub', color='lightyellow', config=config)
+        return None
+
+    releases = response.json()
+    installed_version = config['VERSION']
+    installed_version_parts = parse_tag_name(installed_version)
+
+    # find current version or nearest older version (to link to)
+    current_version = None
+    for i, release in enumerate(releases):
```

Comment:
```suggestion
    for release in releases:
```

You can also use `... in releases.keys():` or `... in releases.values()` if it's a dictionary.